src/generic/video-modal/VideoModal.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { screen } from '@testing-library/react';
+import { render, userEvent, cleanup } from '../../setupTest';
+import { VideoModal } from '.';
+import messages from './messages';
```

Comment:
import ordering: from nearest to furthest

```suggestion
import { render, userEvent, cleanup } from '../../setupTest';
import messages from './messages';
import { VideoModal } from '.';
```

Reply:
let's remove `screen` import, it seems we didn't use it in other tests

Reply:
fixed